java-frontend/src/main/java/org/sonar/java/model/JMethodSymbol.java

```diff
@@ -119,16 +119,6 @@ public TypeSymbol returnType() {
     return thrownTypes;
   }
 
-  @Nullable
-  @Override
-  public MethodSymbol overriddenSymbol() {
-    if (overriddenSymbols == null) {
-      // only compute overridenSymbols once
-      overriddenSymbols();
-    }
-    return firstOverridenSymbol;
```

Comment:
You can also remove the field `firstOverridenSymbol`. Note that this code does raise an issue in SQ, but not in the PR because the line was unchanged...